code/modules/mob/living/carbon/xenomorph/castes/Queen.dm

```diff
@@ -338,13 +337,13 @@
 		/datum/action/xeno_action/onclick/choose_resin/queen_macro, //fourth macro
 		/datum/action/xeno_action/activable/secrete_resin/queen_macro, //fifth macro
 		/datum/action/xeno_action/onclick/grow_ovipositor,
-		/datum/action/xeno_action/onclick/banish,
-		/datum/action/xeno_action/onclick/readmit,
+		/datum/action/xeno_action/onclick/ManageHive,
```

Comment:
Use `snake_case` not `PascalCase`.

See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#type-paths-must-be-snake-case